### PR TITLE
Add reference to rpm packages

### PR DESCRIPTION
**Pull request by jonvanausdeln**
Small update to separate Linux packages into .deb and .rpm flavors, where talking about installers.

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ Positron is built on [Code OSS](https://github.com/microsoft/vscode). To learn a
 
 Install Positron from our [Releases](https://github.com/posit-dev/positron/releases) page on GitHub.
 
-Currently, Positron is producing pre-release builds from a continuous integration (CI) system for macOS, Windows, and Linux. These pre-release builds are tagged with a version number on our Github repository. Select the build you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), or `.deb` (for Linux) file.
+Currently, Positron is producing pre-release builds from a continuous integration (CI) system for macOS, Windows, and Linux. These pre-release builds are tagged with a version number on our Github repository. Select the build you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), `.deb` (for Ubuntu/Debian), or `.rpm` (for Redhat) file.
```
Comment (juliasilge) [inline index.qmd:28]:
```suggestion
Currently, Positron is producing pre-release builds from a continuous integration (CI) system for macOS, Windows, and Linux. These pre-release builds are tagged with a version number on our Github repository. Select the build you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), `.deb` (for Ubuntu/Debian), or `.rpm` (for Red Hat) file.
```

File: start.qmd [hunk 2]
```diff
@@ -58,15 +58,15 @@ install.packages(c("usethis", "cli", "crayon", "rlang", "roxygen2", "pkgload"))
 
 ## Installing Positron
 
-Install Positron from our [Releases](https://github.com/posit-dev/positron/releases) page on GitHub. 
+Install Positron from our [Releases](https://github.com/posit-dev/positron/releases) page on GitHub.
 
-Currently, Positron is producing beta release builds from a continuous integration (CI) system for macOS, Windows, and Linux. Release builds are tagged with a version number on our Github repository. Select the release you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), or `.deb` (for Linux) file.
+Currently, Positron is producing beta release builds from a continuous integration (CI) system for macOS, Windows, and Linux. Release builds are tagged with a version number on our Github repository. Select the release you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), `.deb` (for Ubuntu/Debian), or `.rpm` (for Redhat) file.
```
Comment (juliasilge) [inline start.qmd:63]:
```suggestion
Currently, Positron is producing beta release builds from a continuous integration (CI) system for macOS, Windows, and Linux. Release builds are tagged with a version number on our Github repository. Select the release you want to download, then click on **Assets** and download the `.dmg` (for Mac), `.exe` (for Windows), `.deb` (for Ubuntu/Debian), or `.rpm` (for Red Hat) file.
```

**Review by juliasilge — APPROVED** [review]
Thank you so much @jonvanausdeln!

Just so I know for the future, did you edit the `start.qmd` in visual mode? Wondering about the source of the changes there.

**Review by jonvanausdeln — COMMENTED** [review]
> Thank you so much @jonvanausdeln!
> 
> Just so I know for the future, did you edit the `start.qmd` in visual mode? Wondering about the source of the changes there. 

I accidentally started in visual mode, then switched back.. I'll make sure to not go into visual next time.

**Comment by juliasilge** [comment]
Oh, no worries on that. I should probably set up [canonical mode](https://quarto.org/docs/visual-editor/markdown.html#canonical-mode) for all these `.qmd` files.

**Pull request merged**